ga4gh/gff3.py

```diff
@@ -105,16 +145,20 @@ def __init__(self, fileName=None):
         self.fileName = fileName
         self.roots = set()     # root nodes (those with out parents)
         # index of features by id. GFF3 allows disjoint features with
-        # the same id, although this is rarely used.
+        # the same id.  None is used to store features without ids
         self.byFeatureId = collections.defaultdict(list)
 
     def add(self, feature):
         """
-        add a feature record
+        Add a feature record
         """
+        # None featureId allowed
         self.byFeatureId[feature.featureId].append(feature)
 
     def __linkFeature(self, feature):
+        """
```

Comment:
Did you mean to use a single underscore at the beginning?
http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python

Also, while a docstring isn't necessary in this case:
http://stackoverflow.com/questions/7687407/are-docstrings-for-internal-functions-python-necessary
I'll leave it up to @jeromekelleher to weigh in on whether we should standardize on using one in such a case anyway (it doesn't seem to be forbidden in PEP8 either)